### PR TITLE
Added a VMAP sample app.

### PR DESCRIPTION
**Pull request by pajato1**

### CODE REVIEW

File: AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the catalog object which will start and play the video.
         Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
```
Comment (pajatopmr) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: How does one translate a video of choice into one of these opaque strings?  Is there not a way to provide some sort of id and a string name/id for a video one desires?  Would this be the Wedge API?

Comment (bhnath) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: You can put a string moniker into the strings.xml called 'catalog_api_read_token' or whatever. It's been done for a couple other sample apps.

Comment (pajatopmr) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: Can you give me a reference that I can examine to see if it is what I am looking for?

Comment (bhnath) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: strings.xml:
<resources>
    <string
        name="catalog_api_read_token">"FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA.."</string>
</resources>

MainActivity.java:

Catalog catalog = new Catalog(R.string.catalog_api_read_token);

Comment (pajatopmr) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: Not what I am looking for.  I want the ability to say "give me a catalog
ugly id for the video named 'Return of Sintel', my video cloud id is
"jfafja;lja;rjkl;rejke;lj" or some such.

Comment (bhnath) [inline AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java]: You'd need to do a reverse lookup on the video ID to get the catalog id, which im pretty sure you can't do. Is there another ID to identify your videocloud account other than the catalog id? Wouldn't this 'videocloud' id also be a token and thus messy?

**Pull request merged**